### DISCUSSION TITLE
Admin Panel UI Suggestions

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#33] [Ideas]
Win 10 x64 / Chrome (latest version)

## Breadcrumb Link
**Example**
1. Login to Keygen
2. Click Licenses (or any group)
3. Click an item in the row > opens the detail page
4. Click a related link, i.e., Machines > opens the detail page of the machine

From here, It would be nice if we could just click the breadcrumb in the header to view all the machines (or whatever detail page being viewed).

The above can be reduced to just steps 1-3, where the detail page has the breadcrumbs and the breadcrumb can be clicked to get back to the summary page, instead of having to click the left hand nav, back button, etc...

## Horizontal Scrolling Missing
On several admin summary pages, the data is presented in tabular format. On smaller monitors, i.e. 1920x1080, the data is clipped on the right side and there is no way to view the data without some wonky work-arounds such as highlighting with the mouse to force horizontal scrolling.

**Example**
1. Login to Keygen
2. Click Licenses

**Expected**: can view all the data, including the furthest most right-side data.
**Actual**:  can only view the partial left-side data; the right-side is clipped.

## Open Detail in New Window
**Example**
1. Login to Keygen
2. Click Licenses
3. Right-click a license > open in new tab (or ctrl-click to open in new tab)

**Expected**: License detail opens up in a new tab.
**Actual**: nothing happens.

Would be nice if the summary rows were actual links that could opened in a new tab/window. Currently, you can only click the summary row to get to the detail.

_Thanks for listening._

**Comment by ezekg** [comment]
Appreciate the bug reports and feedback. We don't normally use the dashboard on Windows, so these must be oversights.

We'll work on getting them resolved!
